--- origsrc/gpsd-2.37/Makefile.am	2008-02-17 11:40:47.000000000 -0600
+++ src/gpsd-2.37/Makefile.am	2008-08-26 12:29:52.022242800 -0500
@@ -33,7 +33,7 @@
 if HAVE_PYTHON
 dist_bin_SCRIPTS = gpsprof gpsfake gpscat
 python_PYTHON = gps.py gpsfake.py
-nodist_python_PYTHON = gpspacket.so
+nodist_python_PYTHON = gpspacket.dll
 PYTHONPAGES = gpsprof.1 gpsfake.1 gpscat.1
 endif
 
@@ -173,10 +173,10 @@
 # install may require --root depending on what you're doing.
 # python setup.py install --root=$DESTIR --prefix=$PREFIX
 if HAVE_PYTHON
-PYEXTENSIONS = gpspacket.so
-noinst_SCRIPTS = gpspacket.so setup.py
+PYEXTENSIONS = gpspacket.dll
+noinst_SCRIPTS = gpspacket.dll setup.py
 
-gpspacket.so: gpspacket.c libgps.la
+gpspacket.dll: gpspacket.c libgps.la
 	(pwd="`pwd`"; cd $(srcdir) && $(PYTHON) setup.py build_ext --build-lib "$$pwd" --build-temp "$$pwd/build" --include-dirs "$$pwd")
 endif
 # Clean up after Python
--- origsrc/gpsd-2.37/setup.py	2007-10-17 17:15:07.000000000 -0500
+++ src/gpsd-2.37/setup.py	2008-08-26 12:30:39.819423700 -0500
@@ -11,9 +11,6 @@
 created_files = []
 
 if not 'clean' in sys.argv:
-    if not os.path.exists('gpsd_config.h'):
-	sys.stderr.write('\nPlease run configure first!\n')
-	sys.exit(1)
 
     for f_name in needed_files:
 	if not os.path.exists(f_name):
